src/vsr/message_header.zig

```diff
@@ -1101,10 +1149,11 @@ pub const Header = extern struct {
         size: u32 = @sizeOf(Header),
         epoch: u32 = 0,
         view: u32,
-        version: u16 = vsr.Version,
+        protocol: u16 = vsr.Version,
+        release: u16 = 0, // Always 0.
```

Comment:
Are we going to evict clients based on release? If we are, might be helpful to include replcia's release here?

Reply:
Great idea! Yes, we will evict based on release.